src/runtime-rs/crates/resource/src/cpu_mem/initial_size.rs

```diff
@@ -168,10 +168,15 @@ fn get_nr_vcpu(resource: &LinuxContainerCpuResources) -> u32 {
 
 fn convert_memory_to_mb(memory_in_byte: i64) -> u32 {
     if memory_in_byte < 0 {
-        0
-    } else {
-        (memory_in_byte / 1024 / 1024) as u32
+        return 0;
+    }
+
+    let mem_size = (memory_in_byte / 1024 / 1024) as u32;
```

Comment:
similar as below:
```suggestion
let xxx = 1024 * 1024; // maybe a const is better 
let mem_size = (memory_in_byte / xxx ) as u32;
```

Reply:
It's better to add a comment here to explain the reason for 2M aligned.

for example: the memory size needed to be 2MB aligned for huge page.

Reply:
Done, thanks @lifupan!